pom.xml

```diff
@@ -343,25 +344,8 @@
         </configuration>
       </plugin>
 
-      <!-- JaCoCo for test coverage -->
-      <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>default-prepare-agent</id>
-            <goals>
-              <goal>prepare-agent</goal>
-            </goals>
-          </execution>
-          <execution>
-            <id>default-report</id>
-            <goals>
-              <goal>report</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
+
```

Comment:
Could you format this so that there is only one empty line between different plugins? Just for consistency